ddpui/management/commands/migrate_ui4t_v1_to_v2.py

```
@@ -0,0 +1,228 @@
+                    f"  Found {target_operations.count()} operations for target model"
+                )
+
+                # Create CanvasNodes for operations if they don't exist
+                prev_op_canvas_node: CanvasNode = source_canvas_node
+                for op in target_operations:
+                    op_type = op.config.get("type", "unknown")
+                    other_inputs = op.config.get("config", {}).get("other_inputs", [])
+
+                    other_input_model_nodes: list[SequencedNode] = []
+                    input_models = op.config.get("input_models", [])
+                    for other_input, input_model in zip(other_inputs, input_models[1:]):
```

Ruff (0.14.5)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)